catalog/app/containers/Bucket/PackageTree.js

```diff
@@ -611,13 +618,15 @@ function PackageTree({ bucket, mode, name, revision, path, resolvedFrom }) {
   return (
     <FileView.Root>
       <MetaTitle>{[`${name}@${R.take(10, revision)}/${path}`, bucket]}</MetaTitle>
+      {/* TODO: bring back linked data after re-implementing it using graphql
```

Comment:
Maybe, you wanted to re-implement it in this PR and forgot?

Reply:
i started to migrate that code, but had not enough time to complete it, so it's still a TODO